src/clauses/Call.test.ts

```diff
@@ -29,13 +29,12 @@ describe("CypherBuilder Call", () => {
             .return(movieNode);
         const queryResult = new Cypher.Call(createQuery).build();
         expect(queryResult.cypher).toMatchInlineSnapshot(`
-            "CALL {
-                CREATE (this0:Movie)
-                SET
-                    this0.id = $param0
-                RETURN this0
-            }"
-        `);
+"CALL {
+  CREATE (this0:Movie)
+  SET this0.id = $param0
```

Comment:
This is different with the documentation information, I think one of the two is wrong!